pkg/resources/resource_cluster_replica.go

```diff
@@ -14,6 +15,7 @@ import (
 var clusterReplicaSchema = map[string]*schema.Schema{
 	"name":         ObjectNameSchema("replica", true, true),
 	"cluster_name": ClusterNameSchema(),
+	"comment":      CommentSchema(true),
```

Comment:
Three resources: roles, cluster replicas and indexes are still marked as force new. Breaking out into a followup issue https://github.com/MaterializeInc/terraform-provider-materialize/issues/329